.github/workflows/ci.yml

```diff
@@ -35,6 +35,10 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y cmake libclang-dev libc++-dev gcc-multilib
           cargo build --all-features
+      - name: Build for windows
+        if: matrix.os == 'windows-latest'
+        run: |
+          cargo build --all-features -v
```

Comment:
Why is `-v` used on Windows but not the other platforms?

Reply:
I would suggest removing it here.

If people encounter difficulties on their branches, adding `-v` (or even `-vv` which also includes the CMake output) on their branch can be done fairly quickly.